### DISCUSSION TITLE
Kata1 por que me marca este error por favor si pudieran ayudarme.

### DISCUSSION THREAD

**Discussion by Are31** [LaunchX-InnovaccionVirtual/CursoIntroPython#324] [Q&A]
![image](https://user-images.githubusercontent.com/99683479/154861341-11590352-4c09-4a3b-9338-1086ec893e17.png)

**Comment by GeraMaldonado** [comment]
Buen día, no soy experto, pero espero te sirva, ahi parece que tienes que definir date, creo recordar que mas arriba se importaba una biblioteca:
from datetime import date

Espero haberte podido ayudar, saludos

**Reply by Are31** [reply]
muchas gracias
